packages/replay/src/coreHandlers/handleDom.ts

```diff
@@ -26,8 +39,21 @@ export const handleDomListener: (replay: ReplayContainer) => (handlerData: DomHa
       return;
     }
 
+    const isClick = handlerData.name === 'click';
+    const event = isClick && (handlerData.event as PointerEvent);
+    // Ignore clicks if ctrl/alt/meta keys are held down as they alter behavior of clicks (e.g. open in new tab)
+    if (isClick && slowClickConfig && event && !event.altKey && !event.metaKey && !event.ctrlKey) {
```

Comment:
I changed this to ignore clicks made while alt/meta/ctrl are held down since they can alter behavior that would consider it a dead click.